### PR TITLE
Update the guide for outdated instructions

### PR DESCRIPTION
**Pull request by shin19991207**
Addressed https://github.com/OpenLiberty/guide-cloud-aws/issues/223

### CODE REVIEW

**Review by Ruilin-Ma — APPROVED** [review]
LGTM, thanks!

File: README.adoc [hunk 1]
```diff
@@ -104,9 +102,12 @@ Before you can deploy your microservices, you must create a {kube} cluster.
...
+Before you configure the AWS CLI, you need to create an AWS Identity and Access Management (IAM) user. Navigate to the https://console.aws.amazon.com/iam/home#/users[Identity and Access Management^] users dashboard and create a user through the UI. While creating the user, you must give the user `programmatic access` when selecting the AWS access type. You will be prompted to set permissions for the user. The created user should have the following minimal access levels to perform the guide:
```
Comment (dmuelle) [inline README.adoc]:
```suggestion
Before you configure the AWS CLI, you need to create an AWS Identity and Access Management (IAM) user. Navigate to the https://console.aws.amazon.com/iam/home#/users[Identity and Access Management^] users dashboard and create a user through the UI. When you create the user, select `programmatic access` as the AWS access type. You will be prompted to set permissions for the user. To complete this guide, the created user must have the following minimal access levels:
```

File: README.adoc [hunk 2]
```diff
@@ -104,9 +102,12 @@ Before you can deploy your microservices, you must create a {kube} cluster.
...
+
+* Refer to the https://eksctl.io/usage/minimum-iam-policies/[Minimum IAM Policies^] for the main use cases of `eksctl`.
+* Ensure that the https://docs.aws.amazon.com/AmazonECR/latest/userguide/security-iam-awsmanpol.html#security-iam-awsmanpol-AmazonEC2ContainerRegistryFullAccess[AmazonEC2ContainerRegistryFullAccess^] policy is attached to the IAM account. This policy grants full access to the Amazon Elastic Container Registry, which is necessary for managing container images for your EKS cluster.
 
-You will be prompted for several pieces of information, including an `AWS Access Key ID` and an `AWS Secret Access Key`. These keys are associated with the AWS Identity and Access Management (IAM) user that you created.
+Make sure to take note of the `AWS Access Key ID` and `AWS Secret Access Key`, as you will need these to configure the AWS CLI. After the AWS CLI is installed, configure it by running the `aws configure` command. You will be prompted to provide the `AWS Access Key ID` and `AWS Secret Access Key` associated with the IAM user you created.
```
Comment (dmuelle) [inline README.adoc]:
```suggestion
Make sure to copy the `AWS Access Key ID` and `AWS Secret Access Key` values, as you will need these to configure the AWS CLI. After the AWS CLI is installed, configure it by running the `aws configure` command. You will be prompted to provide the `AWS Access Key ID` and `AWS Secret Access Key` values that are associated with the IAM user you created.
```

**Review by dmuelle — CHANGES_REQUESTED** [review]
ID review: a few minor suggestions

**Review by dmuelle — APPROVED** [review]
LGTM thanks

**Pull request merged**